datalake/src/main/java/com/sequenceiq/datalake/flow/delete/handler/RdsDeletionHandler.java

```diff
@@ -78,20 +93,46 @@ protected Selectable doAccept(HandlerEvent<RdsDeletionWaitRequest> event) {
             LOGGER.error("Database polling exited before timeout. Cause: ", userBreakException);
             response = new SdxDeletionFailedEvent(sdxId, userId, userBreakException, rdsWaitRequest.isForced());
         } catch (PollerStoppedException pollerStoppedException) {
-            LOGGER.error("Database poller stopped for sdx: {}", sdxId, pollerStoppedException);
+            LOGGER.error("Database poller stopped for SDX: {}", sdxId, pollerStoppedException);
             response = new SdxDeletionFailedEvent(sdxId, userId,
                     new PollerStoppedException("Database deletion timed out after " + durationInMinutes + " minutes"),
                     rdsWaitRequest.isForced());
         } catch (PollerException exception) {
-            LOGGER.error("Database polling failed for sdx: {}", sdxId, exception);
+            LOGGER.error("Database polling failed for SDX: {}", sdxId, exception);
             response = new SdxDeletionFailedEvent(sdxId, userId, exception, rdsWaitRequest.isForced());
         } catch (Exception anotherException) {
-            LOGGER.error("Something wrong happened in sdx database deletion wait phase", anotherException);
+            LOGGER.error("Something wrong happened in SDX database deletion wait phase", anotherException);
             response = new SdxDeletionFailedEvent(sdxId, userId, anotherException, rdsWaitRequest.isForced());
         }
         return response;
     }
 
+    private void unscheduleStorageConsumptionCollectionIfNeeded(SdxCluster sdxCluster) {
```

Comment:
I would move all this logic to `ConsumptionService` class.

Reply:
Makes sense. The original `ConsumptionService` has been kept and renamed to `ConsumptionClientService`, and the above logic got moved to `ConsumptionService`.